src/lmic/lmic.c

```diff
@@ -461,7 +461,7 @@ static void reportEventAndUpdate(ev_t ev) {
 }
 
 static void reportEventNoUpdate (ev_t ev) {
-    uint32_t const evSet = 1u << ev;
+    uint32_t const evSet = 1ul << ev;
```

Comment:
1ul is 64-bits on LP64 systems. Instead need to use `UINT32_C(1)`.  (`oslmic_types.h` includes `<stdint.h>`.)